### ISSUE TITLE
Broken link in a notebook

### ISSUE THREAD

**Issue by 0111010001100110** [tensorflow/tensorflow#26470]
Notebook - Intro to CNNs (TF 2.0 Alpha tutorial - images): https://github.com/tensorflow/docs/blob/master/site/en/r2/tutorials/images/intro_to_cnns.ipynb
In the last cell there is a broken link (404): https://github.com/tensorflow/docs/blob/master/site/en/r2/tutorials/_index_/advanced.ipynb ("_As you can see, our simple CNN has achieved a test accuracy of over 99%. Not bad for a few lines of code! For another style of writing a CNN (using the Keras Subclassing API and a GradientTape) head [here](https://github.com/tensorflow/docs/blob/master/site/en/r2/tutorials/_index_/advanced.ipynb)._")

**Comment by mikeza** [comment]
See PR https://github.com/tensorflow/docs/pull/379

Should this be something done in Colab? I saw that had created problems for some people.
